### PR TITLE
Contact Detail page & notification

### LINKED ISSUE #65: Contact Detail Page
There is no page that displays the contact information of the person, only an edit contact page.

PR #63 fixes this.

### PR DESCRIPTION
**Pull request by SachinRammoorthy**
Fixes #65 
Updated notification and created a popup for contact details.

Video demo: https://www.youtube.com/watch?v=v0GybMN-j-M
Task: https://codein.withgoogle.com/dashboard/task-instances/6528904640593920/

### CODE REVIEW

File: app/src/main/res/layout/popup_detail.xml [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+                    android:layout_gravity="center"/>
+
+            </LinearLayout>
+
+        </android.support.v7.widget.CardView>
+
+    </LinearLayout>
+
+
+</LinearLayout>
```
Comment (aashutoshrathi) [inline app/src/main/res/layout/popup_detail.xml]: Add newline at the end

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi updated PR.

**Comment by aashutoshrathi** [comment]
Was there any Issue for this? If yes, add it in PR initiation comment by writing, `Fixes #ISSUE-Number`, else make one.

**Comment by SachinRammoorthy** [comment]
@aashutoshrathi created and referenced issue.

**Pull request merged**